types/theme.d.ts

```diff
@@ -60,6 +60,9 @@ export interface ICSSPseudoElementStyle extends ICSSInJSStyle {
 }
 
 export interface ICSSInJSStyle extends React.CSSProperties {
+  // TODO Questionable: how else would users target their own children?
+  [key: string]: any
```

Comment:
Is it wrong to make this `[key: string]: CssInJssStyle`? I am a bit worried with the any there..

Reply:
Ah, yep. Good call.

Reply:
This is actually quite a bit trickier.  Not every string property is an ICSSInJSStyle.  Some have limited values, such as `speak`, so they cannot be typed as ICSSInJSStyle.

Since this is currently blocking and broken, let's continue with a more permissive object (as is) for now.  We can always improve typings.